test-butler-app/src/main/java/com/linkedin/android/testbutler/SoftKeyboardDisabler.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (C) 2016 LinkedIn Corp.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.android.testbutler;
+
+import android.content.ContentResolver;
+import android.provider.Settings;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+/**
+ * A helper class for disabling or enabling the software keyboard
+ */
+public class SoftKeyboardDisabler {
```

Comment:
I'm having a hard time coming up with a better name for this one, but `SoftKeyboardDisabler` seems misleading. Maybe `ShowImeWithHardKeyboardChanger`? A little verbose but at least it's accurate